code/modules/ai/ai_behaviors/human_mobs/human_mob.dm

```diff
@@ -286,7 +303,7 @@
 	else
 		remove_from_heal_list(old_target)
 	if((human_ai_state_flags & HUMAN_AI_HEALING) && !revive_target)
-		on_heal_end(old_target)
+		on_heal_end(old_target) //this is problematic apparently
```

Comment:
?

Reply:
Forgot to clean it up, note to myself before I added that code above this.